test/addons/node-module-version/test.js

```diff
@@ -4,8 +4,11 @@ const common = require('../../common');
 const assert = require('assert');
 
 const re = new RegExp(
-  'was compiled against a different Node.js version using\n' +
-  'NODE_MODULE_VERSION 42. This version of Node.js requires\n' +
-  `NODE_MODULE_VERSION ${process.versions.modules}.`);
+  `^Error: The module '.+'\n` +
```

Comment:
This PR is just improving the regular expression that matches the existing error message (which includes explicit line breaks). It's not changing the error message itself.